### PR TITLE
Add npm package badge to README.md

### LINKED ISSUE #176: Add npm package badge in Readme .
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update**
 
Let's add badges to the Readme like this : 
Example Readme : https://github.com/publiclab/leaflet-blurred-location-display

[npm version](https://badge.fury.io/js/leaflet-environmental-layers)

Add this line to https://github.com/publiclab/leaflet-environmental-layers/blob/master/README.md : 

```
[![npm version](https://badge.fury.io/js/leaflet-environmental-layers.svg)](https://badge.fury.io/js/leaflet-environmental-layers)
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

### PR DESCRIPTION
**Pull request by ahmadkarlam**
Fixes #176 

I add a npm badge to README

Screen Shot 2019-05-20 at 15 02 10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Leaflet Environmental Layers
+[![npm version](https://badge.fury.io/js/leaflet-environmental-layers.svg)](https://badge.fury.io/js/leaflet-environmental-layers)
```
Comment (sashadev-sky) [inline README.md]:
```suggestion

[![npm version](https://badge.fury.io/js/leaflet-environmental-layers.svg)](https://badge.fury.io/js/leaflet-environmental-layers)
```

**Comment by sashadev-sky** [comment]
@sagarpreet-chadha also if we have the npm badge can we delete from the integrations section of the README:

```
##### NPM :

https://www.npmjs.com/package/leaflet-environmental-layers
```

And then for consistency, open a PR to add a bower badge and delete the bower part too? Does bower have a badge? Never worked with these before

**Comment by ahmadkarlam** [comment]
> Great job here!! Thank you!
> 
> Just for consistency can we add a space between the badge and header? I added a commit suggestion for this :)

Ok, I'll do it.

**Comment by ahmadkarlam** [comment]
> And then for consistency, open a PR to add a bower badge and delete the bower part too? Does bower have a badge? Never worked with these before

Do you mean this? 

Screen Shot 2019-05-20 at 17 39 35

**Comment by sashadev-sky** [comment]
@ahmadkarlam Yes I guess that! I approved here but lets see what @sagarpreet-chadha says because he's the main guy on this repo

**Comment by ahmadkarlam** [comment]
> @ahmadkarlam ...I am going ahead and merging this 🎈 . If you are interested we would love your help in getting bower badge in the Readme.md as well in a separate PR . What do you think ?

I loved too


> This is awesome 👍 😄 !!!
> @sashadev-sky great idea :)
> 
> @ahmadkarlam ...I am going ahead and merging this 🎈 . If you are interested we would love your help in getting bower badge in the Readme.md as well in a separate PR . What do you think ?
> 
> Thank You !

Thanks, I would love too. I will create PR for adding bower badge.

**Pull request merged**